src/components/core/DpEditor/DpEditor.vue

```diff
@@ -416,6 +419,15 @@ export default {
       default: ''
     },
 
+    /**
+     * Needed to get the core_file Route, when DpEditor has imageButton attr.
```

Comment:
Here and below: we should remove the route names from the comments, as they are demosplan-specific.

Reply:
Done in:
https://github.com/demos-europe/demosplan-ui/pull/91/commits/4c1705776448178222bcf3e849148583d3150023
https://github.com/demos-europe/demosplan-core/pull/718/commits/4993a736a4b5019321d421b7fb2be01067a4e631